Content.Server/Gravity/GravityGeneratorComponent.cs

```diff
@@ -145,6 +147,7 @@ private void MakeBroken()
         private void MakeUnpowered()
         {
             _status = GravityGeneratorStatus.Unpowered;
+            EntitySystem.Get<SharedAmbientSoundSystem>().SetAmbience(Owner.Uid, false);
```

Comment:
shouldnt this be getting handled by AmbientOnPowered

Reply:
Wellll gravity generator can be turned off in addition to still being powered.

Reply:
ah, thats silly but out of scope to change